axelor-account/src/main/java/com/axelor/apps/account/service/batch/BatchCloseAnnualAccounts.java

```diff
@@ -104,6 +105,13 @@ protected void start() throws IllegalAccessException {
           batch.getId());
       incrementAnomaly();
       end = true;
+    } catch (PersistenceException e) {
+      TraceBackService.trace(
+          new AxelorException(e, TraceBackRepository.CATEGORY_CONFIGURATION_ERROR, ""),
```

Comment:
Why do you have to transform the PersistenceException in an AxelorException? 
The message of this new AxelorException is empty, at least put the same as the PersistenceException.